src/navigators/PopupWindow.ts

```diff
@@ -31,6 +31,9 @@ export class PopupWindow extends AbstractChildWindow {
         super();
         const centeredPopup = PopupUtils.center({ ...DefaultPopupWindowFeatures, ...popupWindowFeatures });
         this._window = window.open(undefined, popupWindowTarget, PopupUtils.serialize(centeredPopup));
+        if (popupWindowFeatures.closeAutomaticallyPopupWindow) {
+            setTimeout(() => { this.close(); }, popupWindowFeatures.closePopupWindowAfter);
```

Comment:
see https://github.com/authts/oidc-client-ts/pull/798/files

`this.close();` -> `this._abort.raise(new Error("Popup blocked by user"));`
and add a protection around it like: `if (!this._window || this._window.closed) {`